core/storage/translation/gae_models.py

```diff
@@ -34,6 +35,160 @@
 datastore_services = models.Registry.import_datastore_services()
 
 
+# TODO(#14537): This is a duplicate of the same TypedDict in translation_domain,
+# it should be removed after we refactor our importing.
+class TranslatedContentDict(TypedDict):
+    """Dictionary representing TranslatedContent object."""
+
+    content: str|List[str]
+    needs_update: bool
+
+
+class EntityTranslationsModel(base_models.BaseModel):
+    """Model for storing entity translations ."""
+
+    # The id of the corresponding entity.
+    entity_id = datastore_services.StringProperty(required=True, indexed=True)
+    # The type of the corresponding entity.
```

Comment:
This shouldn't be an arbitrary string, right? It should be controlled by a list of choices (and probably its domain object representation should use an enum).

Reply:
Updated.